tlcc_enclave/Makefile

```diff
@@ -6,12 +6,13 @@ TOP = ..
 include $(TOP)/build.mk
 
 PROTO_BUILD_DIR := enclave/protos
+FABRIC_PROTOS := protos/fabric
 BUILD_DIR := _build
 
 .PHONY: protos
 protos: $(PROTO_BUILD_DIR)
 
-$(PROTO_BUILD_DIR):
+$(PROTO_BUILD_DIR): $(FABRIC_PROTOS)
```

Comment:
hmm, wouldn't the dependency have to be rather on the actual proto files than the directory?  (e.g., something along the lines of `$(shell find tlcc_enclave/protos/fabric/ -name '*.proto')`?

Reply:
I add this check to `generate_protos.sh`.

Reply:
hmm, you missed my point: i thought you wanted to add this dependency so generate_protos.sh is always called when protobuf files have changed, not only the very first time when the proto_build_dir does not exist yet (as it will be without any dependency).